File: Sheet/Step 1/Step 1.2/Python/pro1-2-10.py
def star(n):
    star=1
    for i in range(n):
        for j in range(star):
            print('*',end='')
        print()
        star+=1
    star = n-1
    for i in range(n):
        for j in range(star):
            print('*',end='')
        print()
        star-=1

z = int(input())
star(z)


#online Compiler
# def nStarTriangle(n: int) -> None:
#     star = 1
#     for i in range(n):
#         for i in range(star):
#             print("*", end='')
#         print()
#         star=star+1
#     star = n-1
#     for i in range(n):
#         for i in range(star):
#             print("*", end='')
#         print()
#         star=star-1

File: Sheet/Step 1/Step 1.2/C++/pro1-2-10.cpp
#include <iostream>
using namespace std;

int main()
{
    int a = 5;

    for (int row = 0; row < a; row++) // row to less than input
    {
        for (int col = 0; col <= row; col++) // col to row
        {
            cout << "*"; // print star
        }

        cout << endl; // for new line
    }

    for (int row = a; row > 0; row--) // row to 0, post decrementation
    {
        for (int col = 1; col < row; col++) // col to less than row
        {
            cout << "*"; // print star
        }

        cout << endl; // for new line
    }

    return 0;
}
